src/features/Licenses/LicenseDetailTable/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { IntlProvider } from 'react-intl';
+import DataTable from '@edx/paragon/dist/DataTable';
+
+import { columns } from 'features/Licenses/LicenseDetailTable/columns';
+
+const LicenseDetailTable = ({ data, count }) => {
+  const COLUMNS = useMemo(() => columns, []);
+
+  return (
+    <IntlProvider locale="en">
+      <DataTable
+        isSortable
+        columns={COLUMNS}
+        itemCount={count}
+        data={data}
+      >
+        <DataTable.Table />
+        <DataTable.EmptyTable content="No courses found." />
+        <DataTable.TableFooter />
+      </DataTable>
+    </IntlProvider>
+  );
+};
+
+LicenseDetailTable.propTypes = {
+  data: PropTypes.arrayOf(PropTypes.shape([])),
+  count: PropTypes.number,
+};
+
+LicenseDetailTable.defaultProps = {
+  data: [],
+  count: 0,
+};
+
+export default LicenseDetailTable;
```

Comment:
I think this component isn't necessary in master branch now exist the [table](https://github.com/Pearson-Advance/frontend-app-institution-portal/tree/master/src/features/Main/Table) component to avoid to create more files for make more tables.

Reply:
@01001110J Awesome!